elasticdl/python/tests/test_module.py

```diff
@@ -13,6 +13,33 @@ def custom_model():
     return Model(inputs, outputs)
 
 
+def custom_model_with_embedding():
+    inputs = Input(shape=(4,), name="x")
+    embedding = Embedding(4, 2)(inputs)
+    outputs = Dense(1)(embedding)
+    return Model(inputs, outputs)
+
+
+def custom_sequential_model(feature_columns):
+    model = tf.keras.Sequential(
+        [
+            tf.keras.layers.DenseFeatures(feature_columns=feature_columns),
+            tf.keras.layers.Dense(10, activation="relu"),
+            tf.keras.layers.Dense(1, activation="sigmoid"),
+        ]
+    )
+    return model
+
+
+def feature_columns_fn():
+    age = tf.feature_column.numeric_column("age", dtype=tf.int64)
+    education = tf.feature_column.categorical_column_with_hash_bucket(
+        "education", hash_bucket_size=4
+    )
+    education_one_hot = tf.feature_column.indicator_column(education)
+    return [age, education_one_hot]
+
+
```

Comment:
Because we may have many models only for unit tests. Up to now, there are two model files `test_module.py` and `embedding_test_module.py`. 

In my opinion, we can save each model in a separate file, so we can create a new file for these functions.

Or maybe you can take a look at `embedding_test_module.py` to see whether it is convenient to use them.